js/components/report/iframe-answer.tsx

```diff
@@ -41,7 +41,8 @@ export class IframeAnswer extends PureComponent<IProps, IState> {
   }
 
   toggleIframe() {
-    this.setState({iframeVisible: !this.state.iframeVisible});
+    const { iframeVisible } = this.state;
+    this.setState({iframeVisible: !iframeVisible});
```

Comment:
This is fine but just a note you can also use the callback form of `setState`, it works well for toggles:

`this.setState(prev => ({iframeVisible: !prev}))`

Reply:
Thanks 👍 In the latest commit, I made that change (with a slight tweak).